### PR TITLE
PODS-8419 Uncomment email code

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: app/controllers/DeclarationController.scala [hunk 1]
```diff
@@ -54,84 +64,89 @@ class DeclarationController @Inject()(
...
-  private def declarationData(pstr: String, taxYear: TaxYear, loggedInUser: LoggedInUser) = {
+  private def declarationData(pstr: String, taxYear: TaxYear, loggedInUser: LoggedInUser): JsObject = {
 
     val psaOrPsp = loggedInUser.administratorOrPractitioner match {
       case AdministratorOrPractitioner.Administrator => "PSA"
       case AdministratorOrPractitioner.Practitioner => "PSP"
       case _ => throw new RuntimeException("Unknown user type")
     }
 
+    val common = Json.obj(
```
Comment (ngmjohnstone) [inline app/controllers/DeclarationController.scala:122]: Are these changes desirable? I think this was the pipeline breaking issue culprit.

Comment (ngmjohnstone) [reply app/controllers/DeclarationController.scala:122]: Please see declarationDetails used in pipeline fix here: https://github.com/hmrc/pension-scheme-event-reporting-frontend/blob/1a6030a9277626a3d6f35ad06afd97fec6eb35dd/app/controllers/DeclarationController.scala#L104

Comment (grantrobson) [reply app/controllers/DeclarationController.scala:122]: I don't know, this was Shehzad's code. @shehzad-ismail8 any thoughts on this?

**Review by ngmjohnstone — APPROVED** [review]
Looks good to me

**Pull request merged**